topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -170,140 +157,304 @@ To estimate sequence quality along all sequences, we now use [FastQC](https://ww
 >
 {: .hands_on}
 
+> ### {% icon question %} Questions
+>
+> Which Phred encoding is used in the FASTQ file for these sequences?
+>
+> > ### {% icon solution %} Solution
+> > The Phred scores are encoded using `Sanger / Illumina 1.9` (`Encoding` in the top table).
+> {: .solution }
+{: .question}
+
+## Per base sequence quality
+
 Rather than looking at quality scores for each individual read, FastQC looks at quality collectively across all reads within a sample:
 
 ![Per base sequence quality](../../images/quality-control/per_base_sequence_quality.png "Per base sequence quality")
 
-On the x-axis are the base position in the read, and on the y-axis quality scores. In this example, the sample contains reads that are 37 bp long. For each position, there is a box plotted to illustrate the distribution of values (with the whiskers indicating the 90th and 10th percentile scores). The blue line represents the mean score at the postion. The plot background is color-coded to identify good (green), acceptable (yellow), and bad (red) quality scores.
+On the x-axis are the base position in the read. In this example, the sample contains reads that are 37 bp long.
 
-> ### {% icon question %} Questions
+> ### {% icon comment %} Non uniform x-axis
```

Comment:
is this maybe too much info? in the example shown here the axis /is/ uniform, so I would say either leave this out, or show it as a details box with an example image included of a case where this nonuniformity is present

Reply:
Yeah maybe a detail box is better there